extension/src/test/suite/experiments/index.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
```

Comment:
This rule really has some hits and misses. Sometimes we want to list things in a special order (sizes: large, larger, regular, small... number: one, two, three, four...), sometimes it just make sense to write have something come before ({ type: X, payload: Y}, to me it feels more natural to have type come before it limits what can come after). It does not have any say on enums though (because it's linked to a number underneath, I get it), but this make it feel inconsistent. I'd say we remove it / review it (can we enforce after 5 properties?) and use judgment when creating objects, but I don't mind if people want to keep it.

Reply:
It can definitely be annoying. I'll put an item in the retro to discuss, I like the rule because it helps me avoid using smol 🧠 and gives me a few fewer decisions to make in the sea of decisions.